dali/ft/filecopy.cpp

```diff
@@ -2500,14 +2500,20 @@ unsigned FileSprayer::numParallelConnections(unsigned limit)
 
 unsigned FileSprayer::numParallelSlaves()
 {
-    unsigned numPullers = transferSlaves.ordinality();
+    unsigned numPullers = transferSlaves.ordinality();  // == number of targets
     unsigned maxConnections = DEFAULT_MAX_CONNECTIONS;
     unsigned connectOption = options->getPropInt(ANmaxConnections, 0);
+    LOG(MCdebugInfo, job, "In numParallelSlaves():");
+    LOG(MCdebugInfo, job, "    numPullers:%u", numPullers);
+    LOG(MCdebugInfo, job, "    maxConnections:%u", maxConnections);
+    LOG(MCdebugInfo, job, "    connectOption:%u", connectOption);
```

Comment:
Slightly concerned about the extra logging. does it come out in the dfuserver log by default?  If so it is too verbose.

Reply:
I see. I put them in #ifdef _DEBUG .. #endif block.